bh_modules/bashsupport.py

```diff
@@ -8,6 +8,9 @@
 lowercase = import_module("bh_modules.lowercase")
 
 
+BASH_KEYWORDS = ("select", "for", "while", "until")
```

Comment:
Only one blank line is needed before this.  You will only see two before functions and classes in the global scope.  It's a Python PEP8 thing.